samtranslator/open_api/open_api.py

```
@@ -37,10 +37,10 @@ def __init__(self, doc):
         modifications on this copy.
 
         :param dict doc: OpenApi document as a dictionary
-        :raises ValueError: If the input OpenApi document does not meet the basic OpenApi requirements.
+        :raises InvalidDocumentException: If the input OpenApi document does not meet the basic OpenApi requirements.
```

Should this be InvalidDocument or InvalidResource?

---

We can throw `InvalidResource` but we need logical id of the resource.

---

Got it. That gets me thinking but nothing specific for this PR.